web-next/src/routes/(root)/sign/index.tsx

```
@@ -75,6 +83,20 @@ const signCompleteMutation = graphql`
   }
 `;
 
+const signGetPasskeyAuthenticationOptionsMutation = graphql`
```

Biome (2.1.2)

[error] Shouldn't redeclare 'signGetPasskeyAuthenticationOptionsMutation'. Consider to delete it or rename it.

'signGetPasskeyAuthenticationOptionsMutation' is defined here:


(lint/suspicious/noRedeclare)